crates/telio-nurse/src/rtt/ping.rs

```diff
@@ -131,10 +170,12 @@ impl Ping {
                 if let Some(secondary) = t.1 {
                     match secondary {
                         IpAddr::V4(_) => {
-                            dpresults.v4 = Some(ping_action(self.client_v4.clone(), t.0).await);
+                            dpresults.v4 =
+                                Some(ping_action(self.client_v4.clone(), secondary).await);
                         }
                         IpAddr::V6(_) => {
-                            dpresults.v6 = Some(ping_action(self.client_v6.clone(), t.0).await);
+                            dpresults.v6 =
+                                Some(ping_action(self.client_v6.clone(), secondary).await);
                         }
```

Comment:
Can you log in case of None ?

Reply:
Can you elaborate? The ping is logged in `ping_action` closure.

Reply:
I meant for the opposite case to if let Some(secondary) as the second target could be None.

Reply:
Done